otel/otel-configuration-of-collector.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-configuration-of-collector"]
+= Configuration of the OpenTelemetry Collector of the {OTELShortName}
```

Comment:
```suggestion
= Configuring the OpenTelemetry Collector
```
Suggest shorter titles with gerunds where possible

Reply:
This is my personal preference too, but after past feedback from other reviewers, I only use gerunds for procedure content.

Reply:
FYI, the OCP docs guidance (https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings) is as follows:

```
For assemblies and task modules, use a gerund form in headings, such as:
- Creating
- Managing
- Using
```

So, it should be gerunds for assemblies also.

Reply:
If this is the case, then I'm happy to update all headings, but I won't be able to update them for the upcoming release that is next Tuesday. I'm adding your feedback to the Jira epic for the next release.